lib/build/bundler.js

```diff
@@ -233,7 +233,12 @@ exports.Bundler = class {
   }
 
   write() {
-    return Promise.all(this.bundles.map(x => x.write(this.project.build.targets[0])));
+    return Promise.all(this.bundles.map(bundle => bundle.write(this.project.build.targets[0])))
+      .then(async() => {
+        for (let i = this.bundles.length; i--; ) {
+          await this.bundles[i].writeRevToIndex(this.project.build.targets[0]);
```

Comment:
Just realised you did serial write on index file. 👍 
I am not familiar with async/await, took me quite a while to understand what's doing here.